src/app/pages/articles/article-detail/article.component.scss

```diff
@@ -0,0 +1,77 @@
+@import "src/styles/variables";
+@import "src/styles/maps";
+@import "src/styles/mixins";
+
+.article {
+	padding-top: $layout-spacing-large;
+	padding-bottom: $layout-spacing-large;
+	display: flex;
+	flex-flow: column;
+	align-items: center;
+	@include page-styles(standard-spaces);
+
+	&__title-image {
+		max-width: 900px;
+		border-radius: $border-radius-medium;
+		margin-top: $layout-spacing-large;
+		margin-bottom: $layout-spacing-medium;
+	}
+
+	&__title {
+		margin: 0;
+		max-width: 900px;
+		@include headings(h1);
+	}
+
+	&__author {
+		width: 100%;
+		max-width: 700px;
+		display: flex;
+		align-items: center;
+		justify-content: space-between;
+		padding: $layout-spacing-small;
+		border-top: 1px solid $dark;
+	}
+
+	&__content {
+		margin-top: 60px;
+		max-width: 700px;
+		white-space: pre-wrap;
+		@include headings(p2);
+	}
+
+	@include tablet {
+		@include page-styles(tablet-spaces);
+
+		&__title-image {
+			max-width: 700px;
+			margin-top: $layout-spacing-large;
+		}
+		&__title {
+			width: auto;
+			@include headings(h2);
+		}
+		&__content {
+			max-width: 600px;
+			@include headings(p2);
+		}
+	}
+
+	@include mobile {
```

Comment:
Good work 👍 
Maybe you want to adjust the styles in the navbar for mobile as well?